components/Ink/src/MDCInkView.m

```diff
@@ -49,6 +58,12 @@ - (void)commonMDCInkViewInit {
   self.backgroundColor = [UIColor clearColor];
   self.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
   self.inkColor = self.defaultInkColor;
+  _usesLegacyInkRipple = YES;
+}
+
+- (void)setFrame:(CGRect)frame {
+  [super setFrame:frame];
+  self.activeInkLayer.bounds = CGRectMake(0, 0, CGRectGetWidth(frame), CGRectGetHeight(frame));
```

Comment:
Consider moving this to layoutSubviews. This method won't always get called when the view's bounds changes.

Example repro:

```swift

class CustomView: UIView {

  override var frame: CGRect {
    get { return super.frame }
    set {
      super.frame = newValue
    }
  }
}

let testView = CustomView()

// Won't call setFrame:
testView.bounds = .init(x: 0, y: 0, width: 100, height: 100)
```